docs/project-docs/CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
-Contributing Code Changes
+﻿Contributing Code Changes
 =========================
-This page documents the various steps required in order to contribute SparkCLR code changes. 
+This page documents the various steps required in order to contribute Mobius code changes. 
 
 ### Overview
-Generally, SparkCLR uses:
+Generally, Mobius uses:
 * [Github issues](https://github.com/Microsoft/SparkCLR/issues) to track logical issues, including bugs and improvements
 * [Github pull requests](https://github.com/Microsoft/SparkCLR/pulls) to manage the *code review* and merge of *code changes*.
 
 ### Github Issues
```

Comment:
Adding response from CF

---
In reply to: [54300288](https://github.com/Microsoft/SparkCLR/pull/315#discussion_r54300288) [](ancestors = 54300288)